modules/_commands.js

```diff
@@ -53,18 +66,10 @@ module.exports = {
               cmd = cmd.command
             }
             if (typeof cmd === 'function') {
-              var output = cmd(bot, msg)
-              if (!output) {
-                console.log('No return for ' + cmd)
-              } else if (typeof output === 'string') {
-                return bot.say(msg.to, msg.nick + ': ' + output)
-              } else if (output.constructor === Array) {
-                for (var k = 0; k < output.length; k++) {
-                  if (typeof output[k] === 'string') {
-                    return bot.say(msg.to, msg.nick + ': ' + output[k])
-                  }
-                }
+              if (cmd.constructor.name === 'AsyncFunction') {
```

Comment:
`Promise.resolve(cmd(bot, msg))` can be used here to ensure the result is a Promise that you can `.then(...)` even if the `cmd()` returned not-a-promise.